samples/phi3-webgpu/index.html

```diff
@@ -5,9 +5,53 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Phi-3 WebGPU</title>
+    <style>
+      .absolute_center {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+      }
+
+      body {
+        background: rgba(0, 7, 52, 1) url("/assets/bg.png") center / cover
+          no-repeat fixed;
+        overflow-x: hidden;
+        display: flex;
+        flex-direction: column;
+        min-height: 100vh;
+      }
+
+      .main-content {
+        flex: 1;
+      }
+
+      .divider {
+        width: 95%;
+        margin: 10px auto;
+        height: 1px;
+        background-color: rgba(255, 255, 255, 0.2);
+      }
+
+      footer {
+        padding: 0 10px 10px 10px;
+        margin: 0 auto;
+        text-align: center;
+        border-top: 1px solid transparent;
+      }
+    </style>
   </head>
   <body>
     <div id="root"></div>
+    <div class="divider"></div>
+    <footer>
```

Comment:
@NingW101 Do we have common footer for all demos? If so I prefer to add a <footer></footer> tag in each page then insert the nodes by using common script e.g. https://github.com/webmachinelearning/webnn-samples/blob/master/common/component/component.js#L580 to save the maintenance effort. What do you think?

Reply:
Yes, good suggestion, currently we do not have common footer, I'll make footer template and replace them